package.json

```diff
@@ -197,10 +193,6 @@
     "fsevents": "^2.2.1",
     "git-repo-info": "^2.1.1",
     "husky": "^3.1.0",
-    "imagemin-cli": "^6.0.0",
-    "imagemin-jpegtran": "^7.0.0",
-    "imagemin-optipng": "^8.0.0",
-    "imagemin-webpack": "^5.1.1",
```

Comment:
We're on different npm versions:

https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion

```
Note that the file format changed significantly in npm v7 to track information that would have otherwise required looking in node_modules or the npm registry. Lockfiles generated by npm v7 will contain lockfileVersion: 2.

2: The lockfile version used by npm v7, which is backwards compatible to v1 lockfiles.
```

```
$ node -v
v14.13.1
[<_html/wp-content/plugins/google-site-kit]$ npm -v
7.5.6
[<_html/wp-content/plugins/google-site-kit]$ cat .nvmrc
14
```

I went and pulled the old package-lock.json and removed the imagemin packages again and pushed it up, that gets us to `2,716 changes: 133 additions & 2,583 deletions`

Reply:
Thanks! I'm not against using the new lock file format, but we should make this change as a team to avoid back and forth changes to the lock file like this 👍  
I'll open an issue for it.

Reply:
Already opened: https://github.com/google/site-kit-wp/issues/2911